compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -338,7 +341,13 @@ internal class ComposeContainer(
         return if (useSwingGraphics) {
             SwingSkiaLayerComponent(mediator, renderDelegate, skiaLayerAnalytics)
         } else {
-            WindowSkiaLayerComponent(mediator, windowContext, renderDelegate, skiaLayerAnalytics)
+            WindowSkiaLayerComponent(
+                mediator,
```

Comment:
Add explicit parameter names if it's not one-liner

Reply:
Do we add them when variables match parameter names or are obvious?

Reply:
It's better to add it to be on the safe side during possible refactoring